OpenRA.Mods.Common/Traits/Modifiers/FrozenUnderFog.cs

```diff
@@ -64,13 +63,47 @@ public FrozenUnderFog(ActorInitializer init, FrozenUnderFogInfo info)
 			footprint = footprintCells.SelectMany(c => map.ProjectedCellsCovering(c.ToMPos(map))).ToArray();
 		}
 
+		public void Created(Actor self)
```

Comment:
Note that `Created` is *not* equivalent to `AddedToWorld`.  An actor may be added or removed from the world multiple times between its creation and disposal.